packages/plugins/openapi/src/rest-generator.ts

```diff
@@ -868,6 +868,10 @@ export class RESTfulOpenAPIGenerator extends OpenAPIGeneratorBase {
         const required: string[] = [];
 
         for (const field of fields) {
+            if (isForeignKeyField(field)) {
```

Comment:
Hi @thomassnielsen , I believe foreign key fields are actually returned by the RESTAPIHandler today, so changing the spec here will cause inconsistency?

Do you find returning the fk fields to cause trouble?

Reply:
Looks like this only applies to writes. Added a check for mode now.